### DISCUSSION TITLE
Quarto and rmd files are no longer registered by nvim-treesitter

### DISCUSSION THREAD

**Discussion by PMassicotte** [R-nvim/R.nvim#238] [General]
Talking about Quarto YAML completion, there is a new issue: https://github.com/nvim-treesitter/nvim-treesitter/commit/da61d31a3d51f38a78a739392aabf79e7b2f523f

rmd and quarto are no longer registered by nvim-treesitter

**Comment by PMassicotte** [comment]
One way to revert it back:

```lua
vim.treesitter.language.register("markdown", { "quarto", "rmd" })
```

**Comment by jalvesaq** [comment]
Maybe we should do this for users and add an option to disable the register because some users may have a reason not to have Quarto and RMarkdown parsed as Markdown.

**Comment by PMassicotte** [comment]
Should it be done when r-nvim is loading? Read user option and if true register?

**Comment by jalvesaq** [comment]
Yes, I think so.

**Comment by PMassicotte** [comment]
Will make a PR.
